### DISCUSSION TITLE
invalid mesh filename extension 'ata/'

### DISCUSSION THREAD

**Discussion by kerimcharfi** [bulletphysics/bullet3#3864] [Q&A]
I  use this line for creating a MeshCollisionShape from given vertices using pybullet

`stoneId = p.createCollisionShape(p.GEOM_MESH, vertices=vertis,  indices=indices)`

This works most of the times. But some bigger models crash with error.

Any ideas or workarounds? 

### Errormsg

b3Printf: Selected demo: Physics Server
starting thread 0
started MotionThreads thread 0 with threadHandle 00000000000040B4
MotionThreadFunc thread started
Traceback (most recent call last):

 File "****.py", line 91, in <module>
    stoneId = p.createCollisionShape(p.GEOM_MESH, vertices=vertis,  indices=indices)
pybullet.error: createCollisionShape failed.
b3Printf: b3Warning[c:\users\****\appdata\local\temp\pycharm-packaging1\pybullet\examples\sharedmemory\../Importers/ImportURDFDemo/UrdfFindMeshFile.h,54]:

b3Printf: : invalid mesh filename extension 'ata/'

b3Printf: b3Warning[examples/SharedMemory/PhysicsClientSharedMemory.cpp,1338]:

b3Printf: Request createCollisionShape failed
numActiveThreads = 0
stopping threads
Thread with taskId 0 with handle 00000000000040B4 exiting
Thread TERMINATED
finished
numActiveThreads = 0
btShutDownExampleBrowser stopping threads
Thread with taskId 0 with handle 000000000000012C exiting
Thread TERMINATED

**Comment by erwincoumans** [comment]
What is the size of the mesh file? There is a limit to the maximum visual/collision shape.
